app/models/miq_queue_worker_base/runner.rb

```diff
@@ -91,7 +91,7 @@ def get_message
   end
 
   def message_delivery_suspended?
```

Comment:
I think this method can be removed entirely now.

Reply:
Yeah I left it in case there might be a need in the future to do the same thing but could go either way.  I'll just remove it